keyvi/include/keyvi/dictionary/fsa/internal/int_value_store.h

```diff
@@ -41,7 +41,7 @@ namespace internal {
 
 class IntValueStoreBase {
  public:
-  typedef uint32_t value_t;
+  typedef uint64_t value_t;
```

Comment:
makes sense. This limit is probably from the very beginning when keys could only address 4GB.

Can you also change it for `IntInnerWeightsValueStoreBase`?

Reply:
addressed in eb80c2c